playbooks/roles/capsule/tasks/install.yml

```diff
@@ -47,6 +47,11 @@
     src: "/tmp/{{ inventory_hostname }}{{ capsule_certs_tar }}"
     dest: ~/
 
+- name: 'Change cert permissions'
+  shell: chmod 755 -R /etc/pki/katello/private
+  delegate_to: "{{ capsule_server }}"
+  when: devel == true
```

Comment:
A small nitpick for this, you can use the file module instead of chmod... only brought it up because Ansible did first. I think it'd be something along the lines of:
`file: path='/etc/pki/katello/private' mode=0775`

http://docs.ansible.com/ansible/file_module.html